gnucash/gnome/dialog-customer.c

```diff
@@ -917,24 +917,24 @@ gnc_customer_search (GtkWindow *parent, GncCustomer *start, QofBook *book)
     /* Build parameter list in reverse order */
     if (params == NULL)
     {
-        params = gnc_search_param_prepend (params, _("Shipping Contact"), NULL, type,
+        params = gnc_search_param_prepend (params, _("Receiver"), NULL, type,
                                            CUSTOMER_SHIPADDR, ADDRESS_NAME, NULL);
-        params = gnc_search_param_prepend (params, _("Billing Contact"), NULL, type,
+        params = gnc_search_param_prepend (params, _("Person"), NULL, type,
                                            CUSTOMER_ADDR, ADDRESS_NAME, NULL);
-        params = gnc_search_param_prepend (params, _("Customer ID"), NULL, type,
+        params = gnc_search_param_prepend (params, _("Number"), NULL, type,
                                            CUSTOMER_ID, NULL);
-        params = gnc_search_param_prepend (params, _("Company Name"), NULL, type,
+        params = gnc_search_param_prepend (params, _("Name"), NULL, type,
                                            CUSTOMER_NAME, NULL);
     }
 
     /* Build the column list in reverse order */
     if (columns == NULL)
     {
-        columns = gnc_search_param_prepend (columns, _("Contact"), NULL, type,
+        columns = gnc_search_param_prepend (columns, _("Person"), NULL, type,
                                             CUSTOMER_ADDR, ADDRESS_NAME, NULL);
-        columns = gnc_search_param_prepend (columns, _("Company"), NULL, type,
+        columns = gnc_search_param_prepend (columns, _("Name"), NULL, type,
                                             CUSTOMER_NAME, NULL);
-        columns = gnc_search_param_prepend (columns, _("ID #"), NULL, type,
+        columns = gnc_search_param_prepend (columns, _("Number"), NULL, type,
```

Comment:
It can be alphanumeric. E.g. I use their country code as prefix. So ID is more appropriate.